CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 0.55.0 / 2022-03-09
+
+* [CHANGE] Enables 'readonlyRootFilesystem: true' for the Prometheus statefulset. #4566
+* [CHANGE] Add skip TLS verify for the config-reloader HTTP client when informing Prometheus on a config reload. #4592
+* [FEATURE] Add standalone admission webhook. #4494
+* [FEATURE] Support global AlertManagerConfig. #4220
```

Comment:
```suggestion
* [FEATURE] Support the definition of Alertmanager configuration via AlertManagerConfig instead of Kubernetes secret (EXPERIMENTAL). #4220
```